.gitattributes

```diff
@@ -0,0 +1,2 @@
+* text=auto
```

Comment:
This file just says to always normalize line-endings for git in this repo, even if a user's local settings say otherwise.